### ISSUE TITLE
can't find type_traits?

### ISSUE THREAD

**Issue by akohlsmith** [hideakitai/Filters#1]
Using the Filters 0.0.1 installed from the Arduino 1.8.3 "manage libraries" dialog box. My sample sketch can't compile at all. It complains that it can't find "type_traits" which is included in Filters/Helper.h.

Other code builds fine, but I can't include Filters.h without this error.

Example code:
```
#include <Filters.h>

FilterOnePole lpf(LOWPASS, 100.0);

void loop()
{
    lpf.input(analogRead(A0));
}
```

**Comment by akohlsmith-bynorth** [comment]
Same error, 7 months later. I feel there is something obvious I'm doing wrong, but I can't find it. A few google searches show similar errors but no real resolutions. Arduino 1.8.10 on OSX.

**Comment by hideakitai** [comment]
@akohlsmith-bynorth Hi, what kind of board do you use?

**Comment by hideakitai** [comment]
@akohlsmith If I know what board you use, I can support that. If you still want to use this library, reopen and let me know.

**Comment by AurelienConil** [comment]
Arduino 1.8.13 , arduino uno and arduino Mega, same problem : 
`/Users/adminmac/Documents/Arduino/libraries/Filters/Helper.h:5:10: fatal error: type_traits: No such file or directory
 #include <type_traits>
          ^~~~~~~~~~~~~
compilation terminated.
exit status 1
Erreur de compilation pour la carte Arduino Uno`

**Comment by hideakitai** [comment]
Ah, ok. I'll fix it in a few days.

**Comment by hideakitai** [comment]
Fixed and improved everything at version 0.1.0
